app/src/main/res/values-xxhdpi/dimens.xml

```diff
@@ -1,22 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+    <dimen name="small">8dp</dimen>
+    <dimen name="medium">16dp</dimen>
+    <dimen name="large">24dp</dimen>
+    <dimen name="xl">32dp</dimen>
+    <dimen name="xxl">64dp</dimen>
```

Comment:
It seems that this file  has same dimensions as `values/dimens`, is there a reason to keep this file?

Reply:
Removing these configs from xxhdpi is not falling back to values/dimens for some reason as I tested on my real device. I would like to keep these in xxhdpi for safety.